scripts/run-deployment-integration.sh

```diff
@@ -16,7 +16,11 @@ fi
 export NETWORK_NAME=chaintest
 
 sudo ln -sf /usr/src/agoric-sdk/packages/deployment/bin/ag-setup-cosmos /usr/local/bin/ag-setup-cosmos
-rm -rf /usr/src/agoric-sdk/chaintest  ~/.ag-chain-cosmos/ /usr/src/testnet-load-generator/_agstate/agoric-servers/testnet-8000
+# Note: the deployment test and the loadgen test in testnet mode modify these
+# directories, some of them which are in the home directory of the user. While
+# it'd be great if these [tests were more hermetic](https://github.com/Agoric/agoric-sdk/issues/8059),
+# this manual runner must currently reset these paths to ensure reproducible tests.
+rm -rf /usr/src/agoric-sdk/chaintest ~/.agoric/ ~/.ag-chain-cosmos/ /usr/src/testnet-load-generator/_agstate/agoric-servers/testnet-8000
```

Comment:
Easy fix to avoid clobbering the contents of `$HOME`:
```suggestion
# Note: the deployment test and the loadgen test in testnet mode modify these
# directories, some of them which are in $HOME.  We can at least provide an empty $HOME for them.
export HOME="$(mktemp -d deployment-integration-home.XXXXX)"
# While it'd be great if these [tests were more hermetic](https://github.com/Agoric/agoric-sdk/issues/8059),
# this manual runner must currently reset the global paths to ensure reproducible tests.
rm -rf /usr/src/agoric-sdk/chaintest /usr/src/testnet-load-generator/_agstate/agoric-servers/testnet-8000
```

Reply:
That worked, just needed to add `--tmpdir` to make sure it was a fully qualified path in the `/tmp` or wherever `$TMPDIR` is. It has side effects of not clobbering ansible and terraform stuff too, which I prefer.